components/drafts/panel/panel_header.scss

```diff
@@ -0,0 +1,74 @@
+.PanelHeader {
+    display: flex;
+    min-height: 40px;
+    flex-wrap: nowrap;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 16px;
+    border-bottom: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
+    color: rgba(var(--center-channel-color-rgb), 0.64);
+    font-size: 12px;
+    line-height: 16px;
+
+    &__right {
+        display: inline-flex;
+        flex-shrink: 0;
+        flex-wrap: nowrap;
+        align-items: center;
+        justify-content: flex-end;
+
+        div + div {
+            padding-left: 4px;
+        }
+    }
+
+    &__left {
+        display: flex;
+        overflow: hidden;
+        flex-wrap: nowrap;
+        align-items: center;
+        font-weight: 600;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+
+        > span {
+            display: inline-flex;
+            min-width: 0;
+            align-items: center;
+
+            span {
+                overflow: hidden;
+                min-width: 0;
+                text-overflow: ellipsis;
+                white-space: nowrap;
+            }
+        }
+    }
+
+    &__timestamp {
+        color: rgba(var(--center-channel-color-rgb), 0.64);
+        vertical-align: middle;
+    }
+
+    &__sync-icon {
+        display: flex;
+        align-items: center;
+    }
+
+    &__actions {
+        display: none;
+
+        &.show {
+            display: inline-flex !important;
```

Comment:
Assuming we can’t avoid important here.

Reply:
Yeah, not sure there's another way to get the styling we need here.